tests/framework/ROM/TimeSeries/SyntheticHistory/polynomial.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" ?>
```

Comment:
is this file a test? If so it's missing the TestInfo block.

Reply:
fixed